modules/deployment-model/src/test/resources/plugin-successful-structures/plugin-dependencies-no-exportedResources/META-INF/dependencies/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-module-something</artifactId>
+    <packaging>mule-extension</packaging>
+
+    <name>Mule Extension</name>
+    <description>A Mule extension that ..</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-sockets</artifactId>
+            <version>4.0.0</version>
+            <scope>provided</scope>
+            <type>jar</type>
+            <classifier>mule-plugin</classifier>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule</groupId>
+            <artifactId>mule-core</artifactId>
+            <version>4.0.0</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <type>jar</type>
+            <version>1.0.0</version>
+            <artifactId>guava</artifactId>
```

Comment:
Every test pom has dependencies with compile, provided and test scopes. It should also cover system and runtime scopes